packages/application-shell/src/components/custom-view-shell/custom-view-shell.tsx

```diff
@@ -184,58 +185,61 @@ function CustomViewShell(props: TCustomViewShellProps) {
       : hostContext.hostUrl;
 
   return (
-    <ApplicationShellProvider
-      environment={window.app}
-      applicationMessages={props.applicationMessages}
-      apolloClient={props.apolloClient}
-    >
-      {({ isAuthenticated }) => {
-        if (isAuthenticated) {
-          return (
-            <CustomViewContextProvider
-              hostUrl={hostUrl}
-              customViewConfig={hostContext.customViewConfig}
-            >
-              <CustomViewShellAuthenticated
-                dataLocale={hostContext.dataLocale}
-                environment={window.app}
-                messages={props.applicationMessages}
-                projectKey={hostContext.projectKey}
+    <>
+      <GlobalStyles />
```

Comment:
Basically we only include this `GlobalStyles` component which includes a 100% height for the main HTML document content.

Reply:
Ah good catch, thanks!